webapp/views/map/legend.js

```diff
@@ -6,11 +6,13 @@
 
 // Hacking layer controlt, to create a 2nd checkboxot too.
 const layerAddOrigFunc = L.Control.GroupedLayers.prototype._addItem;
+const LEGEND_QUERY = '?SERVICE=WMS&REQUEST=GetLegendGraphic&VERSION=1.0.0&FORMAT=image/png&WIDTH=20&HEIGHT=20&LAYER='
+const RASTER_QUERY = '&LEGEND_OPTIONS=forceRule:True;dx:0.2;dy:0.2;mx:0.2;my:0.2;fontStyle:bold;borderColor:0000ff;border:true;fontSize:10'
```

Comment:
Based on [this post](https://docs.geoserver.org/latest/en/user/services/wms/get_legend_graphic/index.html)